spec/tests/helpers/perf_run_helper_spec.rb

```diff
@@ -793,5 +793,67 @@ class PerfRunHelperClass
       end
     end
   end
+  describe "#save_average_transaction_time" do
+    let(:host) do
+      Beaker::Host.create("localhost", { role: "database" }, logger: @logger)
+    end
+    before { subject.instance_variable_set(:@start_epoch, 0) }
+    before { subject.instance_variable_set(:@end_epoch, 946_684_799) }
+    let(:file) { Tempfile.new(described_class.to_s) }
```

Comment:
Looks like the tests are sharing this file.  So I think that it is possible for one test to read the file the other has written.  That seems bad.

Reply:
This appears to be lazily evaluated prior to each test run, so the file is unique each time.

```
Run options: include {:locations=>{"./spec/tests/helpers/perf_run_helper_spec.rb"=>[796]}}

From: /home/john/Documents/src/jrd-gatling-puppet-load-test/spec/tests/helpers/perf_run_helper_spec.rb @ line 842
:
    841:         binding.pry
 => 842:         file.open
> file
=> #<File:/tmp/PerfRunHelperClass20191126-17557-hqstb2>

From: /home/john/Documents/src/jrd-gatling-puppet-load-test/spec/tests/helpers/perf_run_helper_spec.rb @ line 855
:
    854:         binding.pry
 => 855:         file.open
)> file
=> #<File:/tmp/PerfRunHelperClass20191126-17557-1l9hk7b>
```